python/pyspark/ml/tests/connect/test_connect_evaluation.py

```diff
@@ -20,7 +20,6 @@
 from pyspark.sql import SparkSession
 from pyspark.ml.tests.connect.test_legacy_mode_evaluation import EvaluationTestsMixin
 
-# TODO(SPARK-48084): Reenable this test case
```

Comment:
This test error `pyspark.ml.connect.evaluation not working in 3.5 client <> 4.0 server` is caused by `cloudpickle` forward incompatibility